### ISSUE TITLE
DRY hammer-vlsi unit tests

### ISSUE THREAD

**Issue by jwright6323** [ucb-bar/hammer#231]
_Originally posted by @edwardcwang in https://github.com/ucb-bar/hammer/pull/219_

**Comment by edwardcwang** [comment]
Original comment: HammerSubmitCommandTestContext shares duplicate code with HammerToolHooksTestContext. We don't necessarily need to fix it urgently, although it might be nice to consolidate at some point (large amounts of copy pasted code can get hairy as we all know).

**Comment by colinschmidt** [comment]
Yeah I think we are probably gonna want to make several passes at this. I know last time I added tests I had to some refactoring to avoid too much copy pasting.
